src/fights.ts

```diff
@@ -1282,7 +1282,7 @@ const freeFightSources = [
       CombatLoversLocket.reminisce(monster);
     },
     true,
-    { canOverrideMacro: true }
+    { canOverrideMacro: true, familiar: () => $familiars`Robortender`.find(have) ?? null }
```

Comment:
one small concern here -- does this mean that we're going to potentially ruin our 100% robort drop by forcing robort on a non-black crayon penguin monster? i don't think so but it is probably worth thinking about maybe.

Reply:
It does mean this; I can do a more exhaustive fix later